migrations/5_supplementary.js

```diff
@@ -0,0 +1,35 @@
+const Core = artifacts.require("Core");
+const LinearAuctionPriceCurve = artifacts.require('LinearAuctionPriceCurve');
+const RebalancingTokenManager = artifacts.require('RebalancingTokenManager');
+const SetTokenFactory = artifacts.require("SetTokenFactory");
+
+const ONE_DAY_IN_SECONDS = 86400;
+
+const WBTC_MEDIANIZER_ADDRESS = '';
+const WETH_MEDIANIZER_ADDRESS = '';
+const WBTC_ADDRESS = '';
+const WETH_ADDRESS = '';
+
+module.exports = function(deployer, network, accounts) {
+  if (network == "development" || network == "coverage") {
+    console.log("Exiting - Network is development");
+    return;
+  }
+
+  deployer.then(() => deploySupplmentContracts(deployer, network));
+};
+
+async function deploySupplmentContracts(deployer, network) {
+  // Deploy RebalancingTokenManager
```

Comment:
I'd take it up a notch, right now only deploy in development and include a medianizer. Also include a switch statement and different constants for each environment. Can be addressed in separate PR